providers/implementations/rands/drbg.c

```diff
@@ -389,7 +389,7 @@ int ossl_prov_drbg_instantiate(PROV_DRBG *drbg, unsigned int strength,
         if (drbg->state == EVP_RAND_STATE_ERROR)
             ERR_raise(ERR_LIB_PROV, PROV_R_IN_ERROR_STATE);
         else
-            ERR_raise(ERR_LIB_PROV, PROV_R_ALREADY_INSTANTIATED);
+            ERR_raise(ERR_LIB_PROV, RAND_R_ALREADY_INSTANTIATED);
```

Comment:
That just doesn't seem appropriate ... it is a provider reason error ...

Reply:
Switched this back, I was trying to remove a duplicate error...

I'm assuming that approval is not required again.